src/commands/docs/utils.ts

```diff
@@ -0,0 +1,90 @@
+import { type CommandInteraction, MessageFlags } from 'discord.js';
+import { logToChannel } from '../../util/channel-logging.js';
+import type { ProviderConfig } from './types.js';
+
+export const SEARCH_TERM = '%SEARCH%';
+export const TERM = '%TERM%';
+
+// Utility functions
+export const getSearchUrl = (url: string, search: string) =>
+  url.replace(SEARCH_TERM, encodeURIComponent(search));
+
+export const createBaseConfig = (options: {
+  color: number;
+  icon: string;
+  commandDescription: string;
+  directUrl?: string;
+}) => ({
+  color: options.color,
+  icon: options.icon,
+  commandDescription: options.commandDescription,
+  directUrl: options.directUrl,
+});
+
+export const executeDocCommand = async (
+  config: ProviderConfig,
+  interaction: CommandInteraction
+): Promise<void> => {
+  if (!interaction.isChatInputCommand()) return;
```

Comment:
I know people are a fan of one liner returns, but I find it a lot easier to read when you can read from top to bottom without having to read to the right to understand it. And it's good to be consistent.

```suggestion
  if (!interaction.isChatInputCommand()) {
    return;
  }
```

Reply:
Okay